src/packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -161,7 +164,25 @@ get ${methodName}(): Prisma.${m.model}Delegate;`
 export type RejectOnNotFound = boolean | ((error: Error) => Error)
 export type RejectPerModel = { [P in ModelName]?: RejectOnNotFound }
 export type RejectPerOperation =  { [P in "findUnique" | "findFirst"]?: RejectPerModel | RejectOnNotFound } 
-
+type isTrue<T> = T extends true ? 1 : T extends (err: Error)=> Error ? 1 : 0
```

Comment:
```suggestion
type isTrue<T> = T extends true ? 1 : T extends (err: Error) => Error ? 1 : 0
```

Reply:
And we should call `isTrue` `IsTrue` - uppercase. That's a convention we have in our codebase to distinguish types from the rest.

Reply:
Changing it from isTrue to IsReject